src/main/resources/archetype-resources/silo/src/main/resources/db/common/V1__schema.sql

```diff
@@ -0,0 +1,4 @@
+-- THIS IS AN EXAMPLE - THINK BEFORE YOU USE IT
```

Comment:
I just thought that we could show people how the numbering would be done correctly in this exemple/dummy, but you have to know it anyway before you deploy anything so it's alright.